app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,3 +1,15 @@
+<div class="clearfix">
+  <div class="form-group">
+    <div class="col-md-6">
```

Comment:
is it possible to align the warning with the inputs start?
![Screenshot - 2022-09-11T145344 248](https://user-images.githubusercontent.com/26363699/189526104-538dae87-b877-460e-abf5-a75f4416d2b0.png)